pymmcore_plus/client/_client.py

```diff
@@ -45,7 +45,7 @@ def __init__(
         timeout: int = 5,
         verbose: bool = False,
         cleanup_new=True,
-        cleanup_existing=True,
+        cleanup_existing=False,
```

Comment:
Attention here I think this is important to change, but would be curious what the motivations for having it default to true originally were? Having this be true means that if you start mmcore in one process (jupyter) and then connect and close with another (e.g. napari-micromanger) then the connection to the core will be killed. So i'd be inclined to remove the option entirely.

Reply:
yep i agree, you can default it to `False` and remove the argument.  I think I originally added it because at first I had a lot of stray servers being abandoned, and not being reused.  might have overcompensated :)

Reply:
Removed but also kept the functionality around in `pymmcore_plus.server.try_kill_server` in case it ever comes up in a niche case or when developing.